FlightReservation/src/screens/LogIn.js

```diff
@@ -44,24 +45,23 @@ const [authError, setAuthError] = useState(false);
               <View>
                 <TextField
                   {...formProps}
-                  label={<Span text="email"/>}
+                  label={<Span text="email" />}
                   name="email"
-                  authError = {authError && span('errorAuth')}
-                  />
+                  authError={authError && span('errorAuth')}
+                />
               </View>
               <View>
                 <PasswordField
                   {...formProps}
-                  label={<Span text="password"/>}
+                  label={<Span text="password" />}
                   name="password"
-                  authError = {authError && span('errorAuth')}
+                  authError={authError && span('errorAuth')}
                 />
               </View>
             </View>
             <View>
               <View style={style.buttons_container} />
-              <Pressable
-                onPress={formProps.handleSubmit}>
+              <Pressable onPress={formProps.handleSubmit}>
                 <View style={style.btn}>
                   <Text style={style.button_text}>
                     <Span text="login" />
```

Comment:
this pressable can be implemented as you ButtonForm

Reply:
Done.
We applied the ButtonForm to habilitate/inhabilitate the button.